### PR TITLE
Makefile

### PR DESCRIPTION
**Pull request by LK4D4**
ping @erikh @unclejack

### CODE REVIEW

**Comment by erikh** [comment]
LGTM

File: namespace/namespace_unix.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// build +linux
+// build +unix
```
Comment (mrunalp) [inline namespace/namespace_unix.go]: Shouldn't this be linux?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+PKG:=github.com/docker/docker-network
+PKG_PATH:=/go/src/$(PKG)
+RUN_CMD:=docker run --privileged --rm -v "$(shell pwd)":$(PKG_PATH) -w $(PKG_PATH) golang:1.4 go
+
+build:
```
Comment (mrunalp) [inline Makefile:5]: Hmm, could run the build just locally (not within a container), right?

File: namespace/namespace_unix.go (hunk 1 above)
Comment (erikh) [inline namespace/namespace_unix.go]: yeah I was going to file a PR to fix that. none of this is unix.

File: Makefile (hunk 2 above)
Comment (erikh) [inline Makefile:5]: the problem with running a build locally is that we can't make any guarantees about the environment it'll be built on, hence the container.

Comment (LK4D4) [inline Makefile:5]: @mrunalp You can do locally with just go tools. Makefile will be for docker env.

**Comment by LK4D4** [comment]
@mrunalp Pushed commit with moving to `_linux`. Seems like tag is unneeded if you use `_linux` suffix.

**Comment by unclejack** [comment]
LGTM

**Comment by mrunalp** [comment]
@LK4D4 cool!

**Pull request merged**